packages/grid/x-data-grid/src/hooks/features/preferencesPanel/gridPreferencePanelState.ts

```diff
@@ -2,6 +2,10 @@ import { GridPreferencePanelsValue } from './gridPreferencePanelsValue';
 
 export interface GridPreferencePanelState {
   open: boolean;
+  ids?: {
+    panelId?: string;
+    buttonId?: string;
+  };
```

Comment:
I think we can keep it flat. `labelId` is more generic as it could be used for other things, not only buttons.

```suggestion
  panelId?: string;
  labelId?: string;
```

Reply:
Applied in #8862